File: examples/python/counter.py
import os

counter_file = 'counter.txt'
if os.path.exists(counter_file):
    with open(counter_file) as fh:
        count = int(fh.read())
else:
    count = 0

count += 1
print(count)

with open(counter_file, 'w') as fh:
    fh.write(str(count))

File: examples/r/counter.R
filename = "count.txt"
count = 0

if (file.exists(filename)) {
    as.numeric(readLines(filename)) -> count
}

count <- count + 1

cat(count, end="\n")

fh <- file(filename, "w")
cat(count, end="\n", file = fh)
close(fh)


